doc/src/build/json-rpc.md

```diff
@@ -197,17 +197,19 @@ the [Move](move.md#move) language):
 ```shell
 curl --location --request POST $SUI_RPC_HOST \
 --header 'Content-Type: application/json' \
---data-raw '{ "jsonrpc":"2.0", 
-              "method":"sui_moveCall", 
-              "params":["{{owner_address}}",
-                        "0x2",
-                        "GAS",
-                        "transfer",
-                        [],
-                        ["Pure": "{{coin_object_id_base64}}", "Pure": "{{to_address_base64}}"}],
-                        "{{gas_object_id}}",
-                        2000], 
-              "id":1}' | json_pp
+--data-raw '{ "jsonrpc": "2.0",
+              "method": "sui_moveCall",
+              "params": [
+                  "{{owner_address}}",
+                  "0000000000000000000000000000000000000002",
+                  "ObjectBasics",
```

Comment:
Looks like `GAS` is no longer valid, is `ObjectBasics` the correct move module to use?

Reply:
If we are trying to transfer a coin object, then the module needs to be Coin.

Reply:
I have changed the example to use Coin::transfer_ instead, looks like we have a second transfer(_) in Coin move contract....